docker/nginx/prod.dockerfile

```diff
@@ -3,7 +3,11 @@ FROM nginx:stable-alpine
 ENV TARGET_HOST vor_app
 ENV TARGET_PORT 8080
 
+RUN /ssl/generate.sh ${TARGET_HOST}
```

Comment:
These 3 lines might not be needed for prod since we shouldn't be hardcoding SSL certs into the image. But it is fine for now.

Reply:
should i need to delete the prod dockerfile also? since it won't be used